.github/workflows/ci-tests-mac-os.yml

```diff
@@ -27,7 +27,7 @@ jobs:
           brew reinstall openssl
           export PKG_CONFIG_PATH="/usr/local/opt/openssl@1.1/lib/pkgconfig"
           ./ci/build_picotls.sh
```

Comment:
If you are going to fetch picotls, then you don't need to build it here. Maybe remove this line?

Reply:
thanks .. re-running the commit with the line removed.